### PR TITLE
Add instructions page

### PR DESCRIPTION
**Pull request by MikaelFeher**
Title of the PT story: Instructions page

Link to PT story: https://www.pivotaltracker.com/story/show/128271677

Description of the PR:

1. Adding high_voltage gem for static pages 
2. Adding instructions page with lorem ipsum
3. Adding pages controller with `before action`

screen shot 2016-09-02 at 21 47 58 2

Just because, why not...
screen shot 2016-09-02 at 23 56 08 2

Ready for review!
@tochman @diraoulo

### CODE REVIEW

**Comment by MikaelFeher** [comment]
Created the PR as a "WIP" on account of the feature test only tests the happy path. Although the only sad path I can think of is if the user isn't signed in...

**Comment by diraulo** [comment]
@MikaelFeher this PR should not have commits from your work on the PR Template.

**Comment by MikaelFeher** [comment]
Commits about PR template removed from the PR

**Comment by tochman** [comment]
Conditioning the link to `current_user` is okay but I think we would benefit from generating a PagesController and configure it for HighVoltage. Having that we can add a `before_action :authenticate_user!` and lock pages down that way. Feels more secure (without closing the route a user can still access the url by directly accessing it).
https://github.com/thoughtbot/high_voltage#override

**Comment by MikaelFeher** [comment]
I'll look into that

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class PagesController < ApplicationController
+  include HighVoltage::StaticPage
+  before_action :authenticate_user!
+
+  def show
+    render 'pages/instructions'
+  end
```
Comment (tochman) [inline app/controllers/pages_controller.rb]: Not sure you need this method. I'm worried that you are locking yourself down to only one page. I know we don't have any feature requests for multiple static pages atm, but I would rather have the possibility to be flexible if a need for more pages should arise.

File: config/routes.rb [hunk 2]
```diff
@@ -3,6 +3,8 @@
   devise_for :users
   post 'create_booking', controller: :dashboard, action: :create_booking
   root controller: :dashboard, action: :index
+  resources :pages, only: [:show]
```
Comment (tochman) [inline config/routes.rb]: According to the docs this is the way this route should be configured
```
get "/pages/*id" => 'pages#show', as: :page, format: false
```
but it feels a bit outdated to me, 
However, `resource` and  `resources` feels more for CRUD actions. 
Not sure about this though.

Comment (MikaelFeher) [inline config/routes.rb]: Yeah I saw the docs, but I figured `resources` was a cleaner way to do it

File: app/controllers/pages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  def page_to_render
+    case params[:id]
+    when 'instructions'
+      'instructions'
+    when 'ca_labs'
+      'ca_labs'
+    else
+      raise 'You have not defined the page yet'
+    end
+  end
```
Comment (tochman) [inline app/controllers/pages_controller.rb]: We definitely don't want to raise an error. 
I think you can safely remove the `show` method altogether and use the route setting from the gem docs and let the gem do the magic. ;-)

**Comment by diraulo** [comment]
@tochman @MikaelFeher is this all good now?

**Comment by tochman** [comment]
Still waiting for @MikaelFeher to evaluate my proposed changes

**Pull request merged**